### ISSUE TITLE
Non nodejs install? 

### ISSUE THREAD

**Issue by TashaSkyUp** [KanHarI/gpt-commit-summarizer#29]
Some way to install this in a repo without installing nodejs would be wonderful.. Possible to just fork a repo with it setup already and edit the config? Not really sure about github actions details..

**Comment by KanHarI** [comment]
Yes. See the next section after the NodeJS auto setup, you do not need node js installed.
